cernopendata/modules/fixtures/data/docs/cms-simulated-dataset-names/cms-simulated-dataset-names.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "title": "CMS Simulated Dataset Names",
-    "slug": "cms-dataset-names",
+    "slug": "cms-simulated-dataset-names",
```

Comment:
This was necessary to change in `body.content` too, otherwise the MD file is wrong and the installation breaks... I'll quick-merge a fix. PR #2065